src/CMakeLists.txt

```diff
@@ -268,6 +268,15 @@ elseif ( OS2 )
         VERSION ${LIB_VERSION_INFO}
         SOVERSION ${LIB_VERSION_CURRENT}
     )
+elseif ( WIN32 )
+  set_target_properties ( libfluidsynth
+    PROPERTIES
+      ARCHIVE_OUTPUT_NAME "fluidsynth"
+      PREFIX "lib"
+      OUTPUT_NAME "fluidsynth-${LIB_VERSION_CURRENT}"
```

Comment:
Do we even need to specify `OUTPUT_NAME`, wouldnt `ARCHIVE_OUTPUT_NAME` override it anyway?

Reply:
I think `ARCHIVE_OUTPUT_NAME` applies to both static and shared libraries and then for the shared library file name it is overridden by `OUTPUT_NAME` to add `LIB_VERSION_CURRENT`.

Reply:
If a static build should end up with a `fluidsynth.a` library despite the `PREFIX "lib"` stanza, it might be necessary to change `ARCHIVE_OUTPUT_NAME` to `"fluidsynth"`. Sorry, I am not sure about this anymore since the static library built is disabled by default...

Reply:
> Sorry, I am not sure about this anymore since the static library built is disabled by default...

Nope, just tested and IMHO everything looks good!

This is what gets installed with `-DBUILD_SHARED_LIBS=ON`:
```
lib/libfluidsynth.dll.a
bin/libfluidsynth-1.dll
```
and this is what gets installed with `-DBUILD_SHARED_LIBS=OFF`:
```
lib/libfluidsynth.a
```